### PR TITLE
Update and edit web-worker page

### PR DESCRIPTION
**Pull request by jbogarthyde**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?
Previous attempt got munged when merged (https://github.com/angular/angular/pull/36026)

Issue Number:  https://angular-team.atlassian.net/browse/DOCS-777

## What is the new behavior?
Corrects previously approved content of https://angular.io/guide/web-worker

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: aio/content/guide/web-worker.md [hunk 1]
```diff
@@ -19,9 +19,6 @@ To add a web worker to an existing project, use the Angular CLI `ng generate` co
 You can add a web worker anywhere in your application.
 For example, to add a web worker to the root component, `src/app/app.component.ts`, run the following command.
 
- `ng generate web-worker app`.
```
Comment (gkalpak) [inline aio/content/guide/web-worker.md:22]: Did you mean to remove that? It doesn't look right.

Comment (jbogarthyde) [reply aio/content/guide/web-worker.md:22]: Munged again - fixing it now.

**Comment by kara** [comment]
merge-assistance: was previously approved in https://github.com/angular/angular/pull/36026

File: aio/content/guide/web-worker.md (hunk 1 above)
Comment (gkalpak) [reply aio/content/guide/web-worker.md:22]: This seems to be merged before fixing it.
For reference, I submitted #36397.

**Pull request closed without merging**